lib/elixir/lib/kernel.ex

```diff
@@ -5846,7 +5848,7 @@ defmodule Kernel do
   are inspected. They are the same options accepted by `inspect/2`.
   """
   @doc since: "1.14.0"
-  defmacro dbg(code, options \\ []) do
+  defmacro dbg(code \\ quote(do: binding()), options \\ []) do
```

Comment:
this default is unfortunately not printed nicely in iex:

<img width="354" alt="image" src="https://user-images.githubusercontent.com/76071/180755433-7af7d4d1-059d-439b-b1f8-aca104604128.png">

should we break up docs into dbg/0 and dbg/1,2?

Reply:
We could also make the default be `\\ quoted_binding()` and get better formatting at the cost of "hiding" more?

Reply:
I have no preference here 🙃